### ISSUE TITLE
Minor UI inconsistencies: Input theme references

### ISSUE THREAD

**Issue by timnarr** [microsoft/vscode#93024]
I expected that the overall search input in the UI preferences view is styleable via `settings.textInputBackground` and `settings.textInputForeground`. But currently it is styled via `input.background` and `input.foreground` only.

Please take a look at the attached screenshot. And you may want to use this setting snippet for further debugging.

<img width="1280" alt="vscode-input-theme-references" src="https://user-images.githubusercontent.com/3532570/76908802-e640f580-68a9-11ea-9c7e-c2644cf3dc85.png">

```
    "workbench.colorCustomizations": {
        "input.background": "#0000ff",
        "input.border": "#ffff00",
        "input.foreground": "#ffffff",
        "settings.textInputBackground": "#00ff00",
        "settings.textInputForeground": "#000000",
        "settings.textInputBorder": "#ff0000",
    },
```

**Comment by roblourens** [comment]
Yeah those tokens only apply to setting inputs, that's why there are different ones per setting type. And the search shares the styling of other inputs in vscode, like the search viewlet that you are showing. That seems right? It seems inconsistent that it gets the border token but that was sort of a workaround since the border is needed for many themes to separate it from the background. So you would like it to be themed the same as text setting inputs? I'm not sure, I think I like it being consistent with other workbench inputs since it is not a setting input.

**Comment by TimGr** [comment]
Ah, I see, thank you very much for your explanation. That makes it clearer to me.

> So you would like it to be themed the same as text setting inputs?

Yeah, right, when I read the documentation, I expected to find the reference for the setting-search under "Settings":
https://code.visualstudio.com/api/references/theme-color#settings-editor-colors

My idea was that it might be a good option to have something like "settings.searchBackground" with a fallback to "input.background" if nothing is set. And the same for "foreground" and "border" of course. From my point of view, it would be clearer that way. But after reading your response, I'm not so sure anymore 🤔

**Comment by roblourens** [comment]
Yeah, that's probably a good idea

**Comment by rzhao271** [comment]
It sounds like the intent of textInput colours was to style only setting inputboxes rather than the search box as well.
I'd like to add more documentation on the website to clarify that, and also see whether we can use a different key to style the Settings editor search box border.
